### PR TITLE
Issue 13/getting started

### PR DESCRIPTION
**Pull request by pbaughman**
Documenation-only PR that replaces the top level readme.md and adds a new readme.md for the activities.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,118 @@
 # Kuri EDU
-A minimal kuri robot that exposes kuri's subsystems without any autonomous behavior
+This repository contains software and activities intended to be used for
+educational purposes.  The kuri_edu folder contains a ROS package that provides 
+a minimal running robot framework and the activities folder contains ideas and
+instructions for extending the robot behavior
```
Comment (spmaniato) [inline README.md:5]: * Either here or elsewhere in the main README, you should mention that this assumes a GBZ or EDU binary is already installed. In other words, that the `kuri_edu` package is not self-sufficient. It won't build on an arbitrary computer.
* Maybe tell them that the `kuri_edu` package is already installed in `/opt/edu` along with its dependencies. And that later on they'll be checking out the source code and modifying it.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,118 @@
...
+    * kuri_drive.launch - Runs the hardware abstraction layer and depth-sensor.
+    * madmux_daemon.launch - Provides a hardware abstraction layer to Kuri's camera
+    * vision_bridge.launch - Performs face and object detection
+
+## Getting Started:
+Before attempting any activities, you should make sure you're able to launch the
+kuri EDU software without any modifications.
+
+Start by establishing an SSH connection to Kuri
+`ssh mayfield@kuri-SERIAL_NUMBER`
```
Comment (spmaniato) [inline README.md]: * This is not accurate. The serial number is something like `KR118160000339`. You probably wanna tell them to get the hostname from the mobile app or to construct is as `kuri-LAST_7_DIGITS_OF_SN` (assuming DVT+ robot)
* Connecting to Kuri is gonna be a whole topic in the main `documentation`, so maybe link to that eventually.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,118 @@
...
+
+To set up your shell, source the setup.sh or setup.bash files in the kuri EDU
+package
```
Comment (spmaniato) [inline README.md]: Here you're mixing up the `edu` binary ("GBZ") with the `kuri_edu` package.

File: README.md [hunk 4]
```diff
@@ -1,23 +1,118 @@
...
+the chest LED represents Kuri's battery level.  Green = 100%, Red = 20% or 
+lower.  Kuri should also back up when the bump shell is pressed.
+
+You can stop the kuri_edu software with ctrl-c
 
 ## Keyboard Teleop Demo
 
-To keyboard teleop a Kuri, run kuri_remote.launch or kuri_edu.launch on the robot:
+You can also drive Kuri like a remote control car using the 
+teleop_twist_keyboard node.  You may need to install this node on Kuri.
```
Comment (spmaniato) [inline README.md:68]: They won't have the ROS `apt` sources at this point. You may wanna point to ROS Indigo wiki or our docs. Oh, you've mentioned this below. Maybe say "see below if you are having trouble installing this" here.

File: README.md [hunk 5]
```diff
@@ -1,23 +1,118 @@
...
+Reading package lists... Done
+Building dependency tree       
+Reading state information... Done
+E: Unable to locate package ros-indigo-teleop-twist-keyboard
+```
+
+You may need to add the ROS indigo debian sources to your robot.
+
+```
+sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
```
Comment (spmaniato) [inline README.md:126]: Before this, you need to `sudo mkdir -p /etc/apt/sources.list.d`

**Review by spmaniato — CHANGES_REQUESTED** [review]
@pbaughman See inline for recommended changes

File: README.md (hunk 1 above)
Comment (pbaughman) [reply README.md:5]: Done below

File: README.md (hunk 4 above)
Comment (pbaughman) [reply README.md:68]: Done on line 75

File: README.md (hunk 5 above)
Comment (pbaughman) [reply README.md:126]: Added above

**Review by spmaniato — APPROVED** [review]

**Pull request merged**